### ISSUE TITLE
App won't start

### ISSUE THREAD

**Issue by Dylon0107** [TheOfficialFloW/crazytaxi_vita#11]
I don't know why but my issue was closed before I could get any actual help.

All I get is a black screen then error c2-12828-1 and a link to this file. it was a .psp2dmp file but i changed it to a .txt so i could upload it.

All the files are set up correctly but it still won't start.

any known way to fix this issue?
[psp2core-1648140632-0x00002c20b5-eboot.bin.txt](https://github.com/TheOfficialFloW/crazytaxi_vita/files/8343769/psp2core-1648140632-0x00002c20b5-eboot.bin.txt)

**Comment by dnbass89** [comment]
I get exactly the same issue.

**Comment by Rinnegatamante** [comment]
https://github.com/TheOfficialFloW/crazytaxi_vita/issues/4

**Comment by Dylon0107** [comment]
So I'm assuming at this point the vpk is broken and there is no fix for this issue.

**Comment by Rinnegatamante** [comment]
Yeah the vpk is broken, all the people that made videos about it are all liars.
